Thu Sep 09 2021 22:34:02 GMT-0600 (Mountain Daylight Time)

UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "error".